webpack/scenes/ContentViews/Details/Repositories/ContentViewRepositories.js

```diff
@@ -155,9 +155,11 @@ const ContentViewRepositories = ({ cvId }) => {
   };
 
   const addBulk = () => {
+    setBulkActionEnabled(false);
     setBulkActionOpen(false);
     const reposToAdd = [];
     rows.forEach(row => row.selected && reposToAdd.push(row.repoId));
+    setRows(rows.map(row => ({ ...row, selected: false })));
```

Comment:
Since we refresh the page on add/remove, this is probably not needed?

Reply:
this is what immediately disables the multiple click functionality, and solves that issue.

Reply:
Oh..I thought setBulkActionEnabled(false) and the isDisabled={!bulkActionEnabled} on the button did that.. 😕